AwsCryptographicMaterialProviders/runtimes/python/pyproject.toml

```diff
@@ -0,0 +1,26 @@
+[tool.poetry]
+name = "aws-cryptographic-material-providers"
+version = "0.1.0"
+description = ""
+authors = ["AWS <aws-arg-dafny@amazon.com>"]
+packages = [
+    { include = "aws_cryptographic_materialproviders", from = "src" },
+]
+
+[tool.poetry.dependencies]
+python = "^3.11.0"
```

Comment:
should this instead be the minimum version supported by boto3 and test up to 3.11.0?

Reply:
3.11 is the minimum version supported by Smithy-Python